src/every_query/eval_suite/gen_task.py

```
@@ -0,0 +1,84 @@
+import hashlib
+import re
+from pathlib import Path
+
+import hydra
+import polars as pl
+from omegaconf import DictConfig
+from tqdm import tqdm
+
+
+def code_slug(code: str, n_hash: int = 10, prefix_len: int = 24) -> str:
+    h = hashlib.sha1(code.encode("utf-8")).hexdigest()[:n_hash]
```

Ruff (0.14.14)

[error] Probable use of insecure hash functions in `hashlib`: `sha1`

(S324)